integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDynamicUpdate.java

```diff
@@ -13,6 +13,7 @@
 import java.util.Map;
 import java.util.Set;
 
+import io.kubernetes.client.openapi.ApiException;
```

Comment:
Need to update copyright date given that this is a new file. "2020," becomes "2020, 2021," and "Oracle Corporation" becomes just "Oracle".

Reply:
fixed